docs/source/core_developers_guide/environment_setup.rst

```diff
@@ -144,3 +174,182 @@ install_sphinx
   This plugin installs sphinx, which is required for building the sawtooth
   documentation.
 
+
+Step One: Clone Repository
+==========================
+
+You'll need to have git installed in order to clone the Sawtooth Lake source
+code repository. You can find up-to-date installation instructions here:
+
+* `Git install instructions <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+.. note:: 
+
+  When checking out Sawtooth Lake on Windows for use with vagrant, you should
+  take steps to ensure that Windows-style CRLF line endings are not added to
+  the code. The bash scripts used by your vagrant VM will not run correctly 
+  with CRLF line endings. Git uses a configuration setting, *core.autocrlf*,
+  to control whether or not LF-style line endings are automatically converted
+  to CRLF-style line endings. `This setting should be set in such a way that 
+  CRLFs are not introduced into your repository 
+  <https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration>`_.
+
+Open up a terminal and run the following:
+
+.. code-block:: console
+
+   % cd $HOME
+   % mkdir project
+   % cd project
+   % git clone https://github.com/hyperledger/sawtooth-core.git
+
+.. note::
+
+  On a Windows environment, the suggested version of the last command
+  above is:
+
+  .. code-block:: console
+
+      C:\> git clone https://github.com/hyperledger/sawtooth-core.git
+      --config core.autocrlf=false
+
+
+Step Two: Configure Proxy (Optional)
+====================================
+
+If you are behind a network proxy, follow these steps before continuing:
+
+1. Set the following environment variables:
+
+  * http_proxy
+  * https_proxy
+
+If you are using the Bash shell, run the following commands:
+
+.. warning::
+
+  The example URLs and port numbers used below are examples only.
+  Please substitute the actual URL, with actual port numbers, used
+  in your environment. Contact your network administrator for the
+  information if necessary.
+
+.. code-block:: console
+
+  % export http_proxy=http://example-proxy-server.com:3128
+  % export https_proxy=https://example-proxy-server.com:3129
+
+If you are using Windows, run the following commands:
+
+.. code-block:: console
+
+  % set http_proxy=http://example-proxy-server.com:3128
+  % set https_proxy=https://example-proxy-server.com:3129
+
+
+2. Install the vagrant-proxyconf plugin:
+
+.. code-block:: console
+
+  % cd sawtooth-core/tools
+  % vagrant plugin install vagrant-proxyconf
+
+
+Step Three: Start VM
```

Comment:
I would rather write out _"Virtual Machine"_ here. The verb _"Start"_ also seems a little inaccurate. Maybe, _"Build and Run Virtual Machine"_.